### PR TITLE
[RAPTOR-3968] Allow DRUM multiclass with only 2 labels

### PR DESCRIPTION
**Pull request by meltzg**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Allow drum to run in multiclass mode with only 2 labels

## Rationale
depending on how data is partitioned, it's possible for a partition in multiclass training to be missing labels. Allow DRUM to train on at least 2 labels. Handling only 1 label can/will be done elsewhere.

### CODE REVIEW

**Comment by meltzg** [comment]
jarvis test cmrunner_integration

File: custom_model_runner/datarobot_drum/drum/artifact_predictors/keras_predictor.py [hunk 1]
```diff
@@ -73,12 +73,18 @@ def predict(self, data, model, **kwargs):
...
+                    if len(self.class_labels) > 2:
+                        raise DrumCommonException(
+                            "Target type '{}' predictions must return the "
+                            "probability distribution for all class labels".format(self.target_type)
+                        )
+                    pos_label = self.class_labels[1]
+                    neg_label = self.class_labels[0]
+                else:
+                    pos_label = self.positive_class_label
+                    neg_label = self.negative_class_label
```
Comment (4dahalibut) [inline custom_model_runner/datarobot_drum/drum/artifact_predictors/keras_predictor.py:85]: this hurts me on the inside

File: custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/src/main/java/com/datarobot/custom/ScoringCode.java [hunk 2]
```diff
@@ -52,6 +52,7 @@ public String predict(String inputFilename) throws Exception {
                 csvPrinter = new CSVPrinter(new StringWriter(), CSVFormat.DEFAULT.withHeader(this.classLabels));
                 for (var val : predictions) {
                     HashMap<String, Double> value = (HashMap<String, Double>) val;
+                    System.out.printf("*** %s %s %s\n", this.classLabels, this.positiveClassLabel, this.negativeClassLabel);
```
Comment (4dahalibut) [inline custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/src/main/java/com/datarobot/custom/ScoringCode.java:55]: delet

**Review by 4dahalibut — APPROVED** [review]

**Comment by meltzg** [comment]
jarvis check cmrunner_integration

**Comment by meltzg** [comment]
jarvis test cmrunner_integration

**Review by yakov-g — APPROVED** [review]
When merging, please add: `release` 1.4.5 into the commit message.
And create a tag: v1.4.5

**Comment by datarobot-jarvis** [comment]
:dollar: Total cost of executing tests in Jenkins: $**6.75** USD.

[What is this all about?](http://bit.ly/2BahK4V)

**Pull request merged**